_search-plugins/neural-sparse-search.md

```diff
@@ -55,6 +55,8 @@ PUT /_ingest/pipeline/nlp-ingest-pipeline-sparse
 ```
 {% include copy-curl.html %}
 
+To split long text into paragraphs, use the `text_chunking` ingest processor before the `sparse_encoding` processor. For more information, see [Text chunking processor]({{site.url}}{{site.baseurl}}/ingest-pipelines/processors/text-chunking/).
```

Comment:
Can we be more specific and change `To split long text into paragraphs` to `To avoid information loss by truncating tokens for long documents`? The `text_chunking` processor is a text preprocess step before embedding processor. You can refer to the [RFC](https://github.com/opensearch-project/neural-search/issues/548) for details.